spec/automated_review/empty_database_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe "Empty Database" do
+  it "after migrated remains empty" do
+    counts = ApplicationRecord.connection.tables.each_with_object([]) do |t, array|
```

Comment:
Should we use `ActiveRecord::Base.connection` here? I know we use it in the migrations, but wasn't sure if we should in the specs as well.